sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1091,6 +1100,11 @@ class Analyzer(override val catalogManager: CatalogManager)
           case other => i.copy(table = other)
         }
 
+      case c @ CacheTable(u @ UnresolvedRelation(_, _, false), _, _, _) =>
+        lookupRelation(u.multipartIdentifier, u.options, false)
+          .map(v2Relation => c.copy(table = v2Relation))
```

Comment:
nit: the name should be `relation` instead of  `v2Relation`

Reply:
Thanks, will address in the next PR.